config/config.go

```diff
@@ -60,6 +63,9 @@ func Get() (*Config, error) {
 		KafkaVersion:               "1.0.2",
 		MaxRetries:                 3,
 		MaxSearchResultsOffset:     1000,
+		OTExporterOTLPEndpoint:     "localhost:4317",
+		OTServiceName:              "dp-dimension-search-api",
+		OTBatchTimeout:               5 * time.Second,
```

Comment:
Could the indentation be fixed here please so the "5" lines up with the "dp-..." above.